pom.xml

```diff
@@ -184,19 +184,13 @@
         <dependency>
             <groupId>ch.qos.logback</groupId>
             <artifactId>logback-classic</artifactId>
-            <version>1.2.7</version>
+            <version>1.5.8</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>30.0-jre</version>
-        </dependency>
-        <dependency>
-            <groupId>org.assertj</groupId>
-            <artifactId>assertj-core</artifactId>
-            <version>3.4.1</version>
-            <scope>test</scope>
```

Comment:
I think this should stay at `test` scope

Reply:
I removed it from this package because `assertj-core` is coming from the `logzio-java-sender`:
https://github.com/logzio/logzio-java-sender/blob/master/pom.xml#L150C1-L155C1